x/bank/client/cli_test/helpers.go

```diff
@@ -0,0 +1,46 @@
+package cli
```

Comment:
why not move these tests to `x/bank/client/cli/` and rename this file to `helpers_test.go`?

Reply:
applies to all modules

Reply:
I'm pretty sure this will introduce cyclic dependency issues. Maybe we need to rename, but I think it will break if we move these things back into `.../cli`

Reply:
Given that constraints, I'm not sure this packing name is actually bad

Reply:
How? Adding `_test` is to avoid cyclic dependency issues.

Reply:
But you can't reference the helpers in `_test` from elsewhere. As I understand it the helpers (like bank) are needed for tests in many modules

Reply:
Ahh right, that in of itself will get messy I think, but I suppose we have no choice here. Proceed.